### PR TITLE
Documentation for Coupons

### PR DESCRIPTION
**Pull request by mjfrey**
Initial documentation for Coupon support.

### CODE REVIEW

**Comment by mjfrey** [comment]
@clintonb 
Initial Doc for Coupons.

**Comment by mjfrey** [comment]
@vkaracic @markoivanovic

**Comment by clintonb** [comment]
@srpearce please review.

File: docs/coupons.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
+================
```
Comment (clintonb) [inline docs/coupons.rst]: You don't need the markers above your section headers.

File: docs/coupons.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Coupon Codes
+================
```
Comment (clintonb) [inline docs/coupons.rst]: These should be the length of the text being marked up.

File: docs/coupons.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  Create one or many codes that offer a 100% discount on the cost of a given course
+
+Both types of Coupons can have the following behavior:
+
+  - Once by customer
+
+  - Multiple times by multiple customers
+
+  - Once per customer
```
Comment (srpearce) [inline docs/coupons.rst]: - One time by one user
- Multiple times by multiple users
- One time by multiple users

**Comment by srpearce** [comment]
Just one suggestion to improve clarity.

**Comment by mjfrey** [comment]
@bderusha Please take look.

**Comment by bderusha** [comment]
Looks good to me :+1:

**Pull request merged**